modules/installing-rhv-downloading-rhcos-image.adoc

```diff
@@ -20,22 +20,14 @@ IMPORTANT: Do not substitute the {op-system} image specified by these instructio
 $ mkdir rhcos
 $ cd rhcos
 ----
-
-. In a browser, go to link:https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json[].
-. In `rhcos.json`, find `baseURI` and copy its value.
-. On the Manager, start a `wget` command and paste the value of `baseURI` but *do not press kbd:[Enter]*. For example:
 +
-----
-$ wget https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.4/44.81.202003062006-0/x86_64/
```

Comment:
@dankenigsberg It must use this specific version of rhcos, not the latest version.